tests/Http/CookieTest.php

```diff
@@ -7,6 +7,11 @@
 
 class CookieTest extends TestCase
 {
+	public function tearDown(): void
+	{
+		Cookie::$key = 'KirbyHttpCookieKey';
```

Comment:
Would be safer for future changes if a `setUp` method stores the original key and `tearDown` restores that (whatever it initially was).